pipeline/upi_test_executor.groovy

```diff
@@ -0,0 +1,167 @@
+// Workflow to execute tests
+def sharedLib
+def retVal
+def run_type = "RHCEPH-test-executor run"
+def rhcephVersion= "${params.RHCS_Build}"
+def phase = "${params.Build}"
+def group = "${params.Group}-"
+def suite = "${params.Suite}"
+def buildUserId
+def buildUserEmail
+def buildUserName
+
+def get_ceph_version(){
+    def cephVersion = 'quincy'
+    if(params.RHCS_Build.contains("5.")){
+        cephVersion = 'pacific'
+    }
+    else if(params.RHCS_Build.contains("3.")){
+        cephVersion = 'luminous'
+    }
+    else if(params.RHCS_Build.contains("4.")){
+        cephVersion = 'nautilus'
+    }
+    return cephVersion
+}
+
+def getCLI(){
+    /*
+        Generates the CLI using the arguments provided and returns it.
+    */
+    def cephVersion = 'quincy'
+    if(params.RHCS_Build.contains("5.")){
+        cephVersion = 'pacific'
+    }
+    else if(params.RHCS_Build.contains("3.")){
+        cephVersion = 'luminous'
+    }
+    else if(params.RHCS_Build.contains("4.")){
+        cephVersion = 'nautilus'
+    }
+
+    def cli = "--rhbuild ${params.RHCS_Build}"
+    cli += " --platform ${params.Platform}"
+    cli += " --build ${params.Build}"
+    cli += " --global-conf conf/${cephVersion}/${params.Group}/${params.Conf}.yaml"
+    cli += " --inventory conf/inventory/${params.Inventory}.yaml"
+    cli += " --suite suites/${cephVersion}/${params.Group}/${params.Suite}.yaml"
+    cli += " --cloud baremetal"
```

Comment:
Let's rename it to "upi" from "baremetal".

Reply:
This references at main places in CephCI.

Reply:
ok